brave/mumble/service.py

```diff
@@ -16,9 +16,10 @@
 from datetime import datetime, timedelta
 
 from brave.mumble.auth.model import Ticket
-from brave.api.client import API
+from braveapi.client import API
 
-Ice.loadSlice(b'', [b'-I' + (Ice.getSliceDir() or b'/usr/local/share/Ice-3.5/slice/'), b'Murmur.ice'])
+Ice.loadSlice('-I/usr/share/Ice-3.5.1/slice Murmur.ice')
```

Comment:
Syntax is "path-to-slice-files space ice-file-in-local-directory". It is correct.